cmd/odfcli/set/recovery-profile.go

```diff
@@ -0,0 +1,47 @@
+package set
+
+import (
+	"fmt"
+
+	"github.com/red-hat-storage/odf-cli/cmd/odfcli"
+	"github.com/red-hat-storage/odf-cli/pkg/set"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+var validRecoveryOptions = []string{"high_client_ops", "high_recovery_ops", "balanced"}
+
+var recoveryProfile = &cobra.Command{
```

Comment:
For complex command structures that develop over time, it's good to include the parent in the subcommand name. You can look at Kubectl's command structure, for reference: https://github.com/kubernetes/kubectl/tree/master/pkg/cmd/create

```suggestion
var setRecoveryProfile = &cobra.Command{
```